front/migrations/20241211_parents_migrator.ts

```diff
@@ -89,7 +114,45 @@ const migrators: Record<ConnectorProvider, ProviderMigrator | null> = {
   snowflake: null,
   webcrawler: null,
   zendesk: null,
-  confluence: null,
+  confluence: {
+    transformer: (nodeId, parents) => {
+      // case where we got old IDs exclusively: we add the new IDs to them
+      if (
```

Comment:
In that case why not just do a ...parents, ...parents.map() and then uniq (so that it can be run mulitple times on the same doc?)

Reply:
if there is some kind of inconsistency with what we upsert we won't know about it no?
ofc if i run the script and see assert errors at the first line i'll just change that to what you suggest :p

Reply:
fyi changed all of that to make it idempotent: I turn all old IDs into new ones and then do a unique